bioconda_utils/build.py

```diff
@@ -240,11 +241,15 @@ def build_recipes(
 
     logger.info('blacklist: %s', ', '.join(sorted(blacklist)))
 
+    get_recipes = utils.get_recipes
```

Comment:
Yes, that is right. I did it this way because I thought it better reflects that there is a default case, and a special case (and possibly more in the future).